apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -232,6 +232,19 @@
             WildcardMatcher.valueOf("set-cookie")
         ));
 
+    private final ConfigurationOption<Collection<String>> enabledInstrumentations = ConfigurationOption.stringsOption()
+        .key("enable_instrumentations")
+        .aliasKeys("enabled_instrumentations")
```

Comment:
The Alisas for `disabled_instrumentations` is just here for backward compatibility. I'd suggest not using an alias key for this new option.

```suggestion
```

Reply:
Done